src/components/admin/form-components/SearchBar.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { capFirstLetter, slugify } from 'lib/utilities';
+
+import TextField from 'material-ui/TextField';
+import Menu from 'material-ui/Menu';
+import MenuItem from 'material-ui/MenuItem';
+import ContentAdd from 'material-ui/svg-icons/content/add';
+
+export class SearchBar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: '' };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+    this.props.updateSuggestions(event.target.value);
```

Comment:
Disregard this. I see what is happening. The problem isn't where `value` lives. It's where `suggestions` lives I believe. If I haven't missed anything below, I think the state for suggestions should also live in this component.

Reply:
@frederikbrinck If you change it so that SearchBar owns it's own state you would change the line commented on here from `this.props.updateSuggestions(event.target.value);` => `this.setState(this.props.updateSuggestions(event.target.value));`

It might be worth a rename if you do this to `updateSuggestions` => `getSuggestions`

Reply:
I see what you guys mean. I'll change this :)